Sources/PackageDescription/ReleaseAction.swift

```diff
@@ -0,0 +1,75 @@
+/// Describes individual release action
+public struct ReleaseAction: Equatable, Codable {    
+    /// Release action order
+    public let order: Order
+    
+    /// Action to run
+    public let action: Action
+    
+    init(order: Order,
```

Comment:
❕ I think that you forgot to make this `init` public as otherwise the compiler generated would be sufficient

Reply:
User is supposed to use the `public static func`s (eg `public static func pre(_ predefinedAction: PredefinedAction) -> ReleaseAction`), so this should stay internal.